specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/UpdateWorkspace.json

```diff
@@ -20,6 +20,15 @@
             "740239CE-A25B-485B-86A0-262F29F6EBDB"
           ]
         },
+        "workspaceRepositoryConfiguration": {
```

Comment:
also specify in response

Reply:
Same as above as it is async API call, this workspaceRepositoryConfiguration won't exists here in response.